src/Infrastructure/BotSharp.Core/Agents/Services/AgentService.GetAgents.cs

```diff
@@ -10,40 +11,37 @@ public async Task<List<Agent>> GetAgents()
         var db = _services.GetRequiredService<IBotSharpRepository>();
         var query = from a in db.Agent
                     join ua in db.UserAgent on a.Id equals ua.AgentId
-                    where ua.UserId == _user.Id
+                    join u in db.User on ua.UserId equals u.Id
+                    where u.ExternalId == _user.Id
```

Comment:
Shuld be compatible with `u.id == _user.Id`. 
Reference this fix: https://github.com/SciSharp/BotSharp/blob/2698830c7a3f7046d0959ae2d62b5fa618f54116/src/Infrastructure/BotSharp.Core/Agents/Services/AgentService.GetAgents.cs#L14